### ISSUE TITLE
cssLoader : relative path urlPlugin

### ISSUE THREAD

**Issue by Jiedara** [syscover/pulsar#5]
I, I'm currently using pulsar to make a use of `syscover/CRM` package.
I was struggling around when it came to the login page. The file `/packages/syscover/pulsar/vendor/cssloader/themes/material/elements.html` was returning a 404, and therefor, my login page wasn't working (it was all blank, without any debugging information).

I found that all the URLs called by the cssLoader() weren't customizable, and It was returning to my localhost. 

    $.cssLoader({
            urlPlugin:  'packages/syscover/pulsar/vendor',
            [...]
        });

My Laravel views are currently under `http://localhost/myLaravel/public` and the cssLoader was pointing to http://localhost

To fix it, I had to edit directly the `$cssLoader()` method like this 

    $.cssLoader({
            urlPlugin:  '../../packages/syscover/pulsar/vendor',
            [...]
        });

... Is there a better way to handle this ? Is it possible to make that path customizable ?

TLDR : `localhost/Mylaravel/public/pulsar/pulsar/login` wasn't due to a 404 in the assets, needed to edit a path directly in the `login.blade.php`

**Comment by carlospalacin** [comment]
We have upgrade this package, please install syscover/pulsar package before syscover/crm
Regards!
